File: 1 - Multiples of 3 or 5/solution.py
# Find the sum of all the multiples of 3 or 5 below 1000.
def mutiples(below=1000):
  return sum(
    [
      i
      for i in range(below)
      if i % 3 == 0 or i % 5 == 0
    ]
  )


if __name__ == '__main__':
  sm = mutiples()
  print(sm)

File: 1 - Multiples of 3 or 5/solution.js
// Find the sum of all the multiples of 3 or 5 below 1000.
function mutiples(below=1000) {
  return new Array(below-1)
    .fill(0)
    .map((_,i) => i+1)
    .filter(i => i%3 == 0 || i%5 == 0)
    .reduce((a,c) => a+c, 0);
}

sm = mutiples(10)
console.log(sm)
